x/stakeibc/keeper/msg_server_register_host_zone.go

```diff
@@ -116,11 +116,11 @@ func (k msgServer) RegisterHostZone(goCtx context.Context, msg *types.MsgRegiste
 
 	// add this host zone to unbonding hostZones, otherwise users won't be able to unbond
 	// for this host zone until the following day
-	epochTracker, found := k.GetEpochTracker(ctx, epochtypes.DAY_EPOCH)
+	dayEpochTracker, found := k.GetEpochTracker(ctx, epochtypes.DAY_EPOCH)
 	if !found {
-		return nil, sdkerrors.Wrapf(types.ErrEpochNotFound, "epoch tracker not found, %s", "day")
+		return nil, sdkerrors.Wrapf(types.ErrEpochNotFound, "epoch tracker (%s) not found", epochtypes.DAY_EPOCH)
```

Comment:
sprintf

Reply:
I think this is actually fine since it's Wrapf, not the logger?
https://github.com/Stride-Labs/stride/pull/262/files#diff-025623891f6814e82acc00f5be0799049d46545b47a4643a5f000a88b38adc24R274